android/app/src/main/AndroidManifest.xml

```diff
@@ -34,14 +34,15 @@
         android:theme="@style/AppTheme">
 
         <activity
+                android:exported="true"
                 android:name=".MainActivity"
                 android:configChanges="keyboard|keyboardHidden|orientation|screenSize|uiMode"
                 android:label="@string/app_name"
-                android:launchMode="singleInstance"
+                android:launchMode="singleTask"
```

Comment:
I just need to check this, if i recall correctly, we had a security issue around this...

Reply:
I replaced it.